Source/loadsave.cpp

```diff
@@ -1342,18 +1342,18 @@ static void SavePlayer(SaveHelper *file, int p)
 	file->skip(3); // Alignment
 	file->writeLE<Sint32>(pPlayer->_pSBkSpell);
 	file->writeLE<Sint8>(pPlayer->_pSBkSplType);
-	for (int i = 0; i < 64; i++)
-		file->writeLE<Sint8>(pPlayer->_pSplLvl[i]);
+	for (int8_t spellLevel : pPlayer->_pSplLvl)
+		file->writeLE<Sint8>(spellLevel);

```

Comment:
is there a reason why some places are Sint8 and others became int8_t?

Reply:
It's two different things. Again will be cleaned up later. We already explained this yesterday :P